Application/EdFi.Ods.Admin.Api/Compose/pgsql/env.example

```diff
@@ -1,24 +1,25 @@
-API_MODE=<API Mode Eg. SharedInstance, YearSpecific, DistrictSpecific>
-ADMIN_API_VIRTUAL_NAME=<virtual name for the Admin API endpoint>
-ODS_VIRTUAL_NAME=<virtual name for the ods endpoint>
+API_MODE=SharedInstance
+ADMIN_API_VIRTUAL_NAME=adminapi
+ODS_VIRTUAL_NAME=webapi
```

Comment:
I changed this file to have sensible default values instead of making the user guess what values to enter.